code/modules/paperwork/folders.dm

```diff
@@ -6,22 +6,27 @@
 	w_class = ITEM_SIZE_SMALL
 
 /obj/item/weapon/folder/blue
+	name = "blue folder"
 	desc = "A blue folder."
 	icon_state = "folder_blue"
 
 /obj/item/weapon/folder/red
+	name = "red folder"
 	desc = "A red folder."
 	icon_state = "folder_red"
 
 /obj/item/weapon/folder/yellow
+	name = "yellow folder"
 	desc = "A yellow folder."
 	icon_state = "folder_yellow"
 
 /obj/item/weapon/folder/white
+	name = "white folder"
 	desc = "A white folder."
 	icon_state = "folder_white"
 
 /obj/item/weapon/folder/nt
```

Comment:
NT does not exist in the Atlas setting. While removing NT is a long process, not adding new references is the first step. A handful of folders with logos won't go amiss, once we determine them.

Reply:
Regarding Nanotrasen, of course they don't exist in our setting, and we won't use their assets. But I don't think we should remove them from the codebase
all it will do is break compatibility with bay, and make updating harder
Regarding that specific folder, it already existed, i just added a name variable to it for completeness' sake